scout/adapter/mongo/panel.py

```diff
@@ -344,7 +344,7 @@ def add_pending(self, panel_obj, hgnc_gene, action, info=None):
             updated_panel(dict):
 
         """
-        
+
```

Comment:
You can move indentation for the code block after `if action ...` back to make it more flat. We know at that stage that the action is valid

Reply:
I'm not sure that this change won't result in a bug, have you tested it?